test/test_moc_globe_atl.py

```diff
@@ -65,70 +72,64 @@ class Test_Moc_Globe_Atl(ut.TestCase):
 
     def test_moc_globe_atl_float64(self):
 
-        out_arr = geocat.f2py.moc_globe_atl(lat_aux_grid,
-                                            tmp_a.astype(np.float64),
-                                            tmp_a.astype(np.float64),
-                                            tmp_a.astype(np.float64), t_lat,
-                                            rmlak)
+        out_arr = moc_globe_atl(lat_aux_grid, tmp_a.astype(np.float64),
+                                tmp_a.astype(np.float64),
+                                tmp_a.astype(np.float64), t_lat, rmlak)
 
         nt.assert_array_almost_equal(ncl_truth, out_arr)
         nt.assert_equal((3, 2, kdep, nyaux), out_arr.shape)
 
     def test_moc_globe_atl_float64_xr(self):
 
-        out_arr = geocat.f2py.moc_globe_atl(
-            xr.DataArray(lat_aux_grid), xr.DataArray(tmp_a.astype(np.float64)),
-            xr.DataArray(tmp_a.astype(np.float64)),
-            xr.DataArray(tmp_a.astype(np.float64)), xr.DataArray(t_lat),
-            xr.DataArray(rmlak))
+        out_arr = moc_globe_atl(xr.DataArray(lat_aux_grid),
+                                xr.DataArray(tmp_a.astype(np.float64)),
+                                xr.DataArray(tmp_a.astype(np.float64)),
+                                xr.DataArray(tmp_a.astype(np.float64)),
+                                xr.DataArray(t_lat), xr.DataArray(rmlak))
```

Comment:
Another linter change I don't like. Prev version was cleaner and more compact.

Reply:
OBE

Reply:
Here is how it looked like in `main` branch:

```
        out_arr = geocat.f2py.moc_globe_atl(
            xr.DataArray(lat_aux_grid), xr.DataArray(tmp_a.astype(np.float64)),
            xr.DataArray(tmp_a.astype(np.float64)),
            xr.DataArray(tmp_a.astype(np.float64)), xr.DataArray(t_lat),
            xr.DataArray(rmlak))
```

I don't agree that it was cleaner and compact because it did not look consistent as the rest of the code (e.g. Lines 90-94) had the function arguments just aligned to each other:

```
out_arr = geocat.f2py.moc_globe_atl(lat_aux_grid,
                                    tmp_a.astype(np.float32),
                                    tmp_a.astype(np.float32),
                                    tmp_a.astype(np.float32), t_lat,
                                    rmlak)
```

This is more of a personal preferences difference though, and I could be fine with either way.